flambe/experiment/experiment.py

```diff
@@ -342,10 +344,17 @@ def trial_name_creator(trial):
                                                   raise_on_failed_trial=False)
                 logger.debug(f"Finish running all tune.Experiments for {block_id}")
 
+                any_error = False
                 for t in trials:
                     if t.status == t.ERROR:
                         logger.error(f"{t} ended with ERROR status.")
                         success[block_id] = False
+                        any_error = True
+                if any_error and self.stop_on_failure:
+                    self.teardown()
+                    logger.error(f"Stopping experiment at block '{block_id}' because there was an "
```

Comment:
we could add colors here. `cl.RE` for red one.

Reply:
Check this line https://github.com/asappresearch/flambe/pull/106/files#diff-7450e432b0e35241531af760c8a6d4eaR410

for reference

Reply:
`self.teardown()` updates the status I think this is okay